Makefile

```
@@ -26,7 +26,7 @@ STYLE_CHECK_DIRS := src demos/common/cpp/src demos/image_classification/cpp demo
 HTTP_PROXY := "$(http_proxy)"
 HTTPS_PROXY := "$(https_proxy)"
 NO_PROXY := "$(no_proxy)"
-JOBS ?= $(shell python3 -c 'import multiprocessing as mp; print(mp.cpu_count())')
+JOBS ?= $(shell python3 -c 'import multiprocessing as mp; print(int(mp.cpu_count()/2))')
```

```suggestion
JOBS ?= $(shell python3 -c 'import psutil; print(psutil.cpu_count(logical=False))')
```
Otherwise this won't work properly on machines with disabled HT.